daemon/src/payout_curve/csr_tools.rs

```diff
@@ -48,6 +48,9 @@ impl CSR {
     // matrix version of `solve()`; useful for solving AX = B. Implementation
     // is horrible.
     pub fn matrix_solve(&self, b_arr: &Array2<f64>) -> Result<Array2<f64>, Error> {
+        if self.is_square() == false {
```

Comment:
```suggestion
        if !self.is_square() {
```

A bit more idiomatic I'd say.